site/src/routes/metrics.js

```diff
@@ -0,0 +1,155 @@
+import { stores } from '../constants.js'
+import { get as getDeals } from '../models/deals.js'
+
+const MAX_AGE_SECS = 600 // max age of a metrics response in seconds
+const STALE_WHILE_REVALIDATE_SECS = 3600
+const CACHE_KEY = '/metrics'
```

Comment:
```suggestion
const nanoid = require('nanoid/non-secure')
const cacheBust = nanoid.nanoid()
const CACHE_KEY = '/metrics' + cacheBust
```

the cache bust is important, this will invalidate all caches when we deploy a new worker and avoid issues if the new worker changes the responses

Reply:
Right but, do we need a whole a module to generate an universally unique id? Why not just add a suffix like `/metrics-1` iff we change the code? We only really want to invalidate the cache if we change the code right?

Reply:
Wouldn't this generate a new code each time a worker is run?

Reply:
yes sure, the suggested change would just make this automatic. Im fine with both.

Reply:
no, the value of `cacheBust` is kept between worker runs. Each time we deploy a v8 isolate is created and keeps that value stable until a new deploy is made.

Reply:
Ok well, I think we should just keep it simple for now. Given what this is being used for and the short time involved I can't imagine this being a huge problem even if there were no busting...